specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/computeRPCommon.json

```diff
@@ -1769,6 +1769,10 @@
             "name": "WindowsPatchAssessmentMode",
             "modelAsString": true
           }
+        },
+        "automaticByPlatformSettings": {
+          "$ref": "#/definitions/WindowsAutomaticByPatchSettings",
+          "description": "Specifies additional settings for patch mode AutomaticByPlatform in VM Guest Patching on Windows."
```

Comment:
Does this only apply to installation or to all AutomaticByPlatform operations?

Reply:
Right now the payload only targets patching, but can just as easily support assessment related parameters too if/when suitable ones are added.